rebench.conf

```diff
@@ -205,6 +205,45 @@ benchmark_suites:
                 extra_args: 250
                 codespeed_name: "[shoot] specnorm_n"
 
+    simple:
```

Comment:
There is an experiment block below with an experiment named AreWeFast. The block specifies the benchmark run for that experiment. AreWeFast is the experiment we run in the benchmarking machine. So, you have to add the simple benchmarks there also.